### DISCUSSION TITLE
Prevent GitHub release changelog overwriting

### DISCUSSION THREAD

**Discussion by srnyx** [Kira-NT/mc-publish#82] [Q&A] [answered]
When I publish a GitHub release mc-publish overwrites the GitHub release's changelog when adding the files to the release, how can I prevent this?

**Comment by Kira-NT** [comment]
As per the documentation, the `changelog` field is, by default, set to the "body of the release that triggered the action". This essentially means that `mc-publish` might be overwriting the GitHub Release's changelog with the same changelog. However, this would typically not result in any changes if these changelog versions are the same. Thus, I'm wondering if there could be some other variables in your configuration that may be causing this issue.

Could you provide more details about the specific configuration you're using, or perhaps an example where this is happening? With more context, it would be easier to diagnose the exact issue and suggest a workaround or solution.

**Reply by srnyx** [reply]
Yeah my `changelog` value is set to `${{ github.event.release.html_url }}` (https://github.com/srnyx/tools/blob/main/.github/workflows/publish.yml#L64)

**Reply by Kira-NT** [reply, marked as answer]
Ah, I see. In this case you can utilize `github-changelog` to keep the original changelog intact, as shown in #76.
